flight/PiOS/STM32/pios_servo.c

```diff
@@ -598,6 +599,20 @@ void PIOS_Servo_SetFraction(uint8_t servo, uint16_t fraction,
 	PIOS_Servo_SetRaw(servo, val);
 }
 
+bool PIOS_Servo_IsDshot(uint8_t servo) {
```

Comment:
Let's us force dshot-style scaling in actuator irrespective of min/max.  (And handle the weird 3D map)